gate-web/gate-web.gradle

```diff
@@ -50,6 +50,10 @@ dependencies {
 
   implementation "io.springfox:springfox-swagger2"
 
+  implementation "io.cloudevents:cloudevents-spring:2.3.0"
+  implementation "io.cloudevents:cloudevents-json-jackson:2.3.0"
+  implementation "io.cloudevents:cloudevents-http-basic:2.3.0"
```

Comment:
Appreciate this PR has been open a while, but the latest versions appear to be `2.5.0`?